modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,94 @@
+# Candidate Parameters
+
+## Purpose
+
+The Candidate Parameters module allows users to input 
+candidate data that is not timepoint specific.
+
+## Intended Users
+
+The primary types of users are:
+1. Data entry personnel
+2. Site coordinators validating the data
+
+## Scope
+
+The Candidate Parameters module stores all information 
+for a candidate that are not timepoint dependent. All data 
```

Comment:
is not

Reply:
done

Reply:
seems to still say "are not"? Did you forget to push it?

Reply:
lol I just pushed I was just confirming that i covered the 99 comments on that cursed readme

Reply:
wait .. so are you gonna change this? lol

Reply:
it should already be done, check files changed

Reply:
still says "are", look at the diff..